diff --git a/dist/constants.js b/dist/constants.js
index 973e2cbcd6ea11514cb7971cdf3eec2e664e0855..9f89c92e59c564e8650ab87e28ec4953aef1a8f5 100644
--- a/dist/constants.js
+++ b/dist/constants.js
@@ -4,8 +4,8 @@ exports.DEFAULT_JEST_TEST_MATCH = exports.JS_EXT_TO_TREAT_AS_ESM = exports.TS_EX
 exports.LINE_FEED = '\n';
 exports.DECLARATION_TYPE_EXT = '.d.ts';
 exports.JS_JSX_EXTENSIONS = ['.js', '.jsx'];
-exports.TS_TSX_REGEX = /\.tsx?$/;
-exports.JS_JSX_REGEX = /\.jsx?$/;
-exports.TS_EXT_TO_TREAT_AS_ESM = ['.ts', '.tsx'];
+exports.TS_TSX_REGEX = /\.m?tsx?$/;
+exports.JS_JSX_REGEX = /\.m?jsx?$/;
+exports.TS_EXT_TO_TREAT_AS_ESM = ['.ts', '.tsx', '.mts'];
 exports.JS_EXT_TO_TREAT_AS_ESM = ['.jsx'];
 exports.DEFAULT_JEST_TEST_MATCH = ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'];
